native/cocos/renderer/pipeline/forward/ForwardStage.cpp

```diff
@@ -90,7 +90,7 @@ void ForwardStage::destroy() {
     CC_SAFE_DELETE(_batchedQueue);
     CC_SAFE_DELETE(_instancedQueue);
     CC_SAFE_DELETE(_additiveLightQueue);
-    CC_SAFE_DELETE(_planarShadowQueue);
+    CC_SAFE_DESTROY_AND_DELETE(_planarShadowQueue);
```

Comment:
As `destructor` invokes `destroy()`, so can just use `CC_SAFE_DELETE`.

Reply:
May be can just remove `destroy()` if it can not be reused.

Reply:
I think we could keep the logic of destroy here now since it's the same behavior as which in ts.
And there are other code also invoke destroy:
<img width="649" alt="WeChat8d3469c0f8b05f1716ffb420b22ec569" src="https://user-images.githubusercontent.com/493372/180952133-b2a5fdca-ea06-4a57-829c-b36318e734b1.png">

If we need to make destroy private, let's do that change in another PR ?

Reply:
Yep, can do in another PR.